app/packs/src/decidim/vocdoni/admin/utils/create_vocdoni_election.js

```diff
@@ -2,7 +2,7 @@ import { Election, PlainCensus } from "@vocdoni/sdk";
 import { initVocdoniClient } from "src/decidim/vocdoni/admin/utils/init_vocdoni_client";
 
 // Allow the user to vote multiple times
```

Comment:
As we're changing the value for vote overwrite, we need to update accordingly the comment:

```suggestion
// Do not allow the user to vote multiple times
```